wasm2c/wasm-rt.h

```diff
@@ -242,6 +250,11 @@ extern void wasm_rt_allocate_table(wasm_rt_table_t*,
                                    uint32_t elements,
                                    uint32_t max_elements);
 
+/**
+ * Free a Table object.
+ */
+extern void wasm_rt_free_table(wasm_rt_table_t*);
```

Comment:
Perhaps the freeing of tables and memories could also be introduced as a separate PR before this one lands?   Its seems unrelated to instancing.

Reply:
Sure, we will do that. It's eventually required because otherwise asan can complain about the memory leak.

Reply:
Done in #1901.